Tools/SwiftLint.md

```diff
@@ -35,37 +35,19 @@ You can find all rules by running `$ swiftlint rules`.
 
 You have the possibility to control which rule is disabled/enabled and set thresholds for warnings and errors for a given rule. Just create a `.swiflint.yml` file and add it to your project root directory.
 
-To disable a rule, add the following to your .yml file:
+You need to change the `ProjectNameTests`  to the name of your Tests folder in the Project.
 
-```yaml
-disabled_rules: # rule identifiers to exclude from running
-  - force_cast
-  - force_unwrappingitespace
-  - type_name
-```
-
-To customize a rule threshold for warnings and errors, you can add one of the following:
-
-```yaml
-force_unwrapping: warning
-
-line_length: 510
+To disable a rule, remove it from the list of  `only_rules` and it will not be included in the linting process.
```

Comment:
```suggestion
To disable a rule, comment it out from the list of `only_rules` and it will not be included in the linting process. However, if you do so, leave a comment why the rule is disabled for future reference and as a reminder that in the end the project should be updated to accommodate the new rules.
```